contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -423,6 +423,11 @@ contract TrueCreditAgency is UpgradeableClaimable, ITrueCreditAgency {
             _enterDefault(pool, borrower, DefaultReason.InterestOverdue);
             return;
         }
+        uint256 overBorrowLimit = overBorrowLimitTime[pool][borrower];
```

Comment:
I wouldn't create a separate variable for it, especially because its name can be confused with asset amount that is over limit. Also it is used only once in the following if statement.

Reply:
It would save some gas, because hash function will be called fewer times. Maybe more descriptive name would do? Something like `_overBorrowLimitTime`?